### ISSUE TITLE
`TypeError: Cannot read property 'type' of undefined`

### ISSUE THREAD

**Issue by ljharb** [semantic-release/cracks#5]
I did `npm install -g cracks` and then in a project, `cracks`, and got:

```
"$(npm root -g)"/cracks/dist/index.js:35
  var type = nextRelease.type || opts.type;
                        ^

TypeError: Cannot read property 'type' of undefined
    at exports.default ("$(npm root -g)"/cracks/dist/index.js:35:25)
    at exports.default ("$(npm root -g)"/cracks/dist/cli.js:40:21)
    at Object.<anonymous> ("$(npm root -g)"/cracks/bin/cracks:3:26)
    at Module._compile (module.js:398:26)
    at Object.Module._extensions..js (module.js:405:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:430:10)
    at startup (node.js:141:18)
    at node.js:980:3
```

I also tried with `-p package.json,test` etc and got the same thing.

**Comment by Guria** [comment]
confirming

**Comment by Guria** [comment]
Confirming.
I want to use cracks as posttest script. It would prevent not only wrong publish but even merge to master because of failed build. But currently it is not working from command line properly.

**Comment by boennemann** [comment]
ping @christophwitzko

**Comment by christophwitzko** [comment]
Hi,
thank you for reporting the issue. It should be fixed in `cracks@3.1.2`.
Have a great day,
Christoph
